zap/src/main/java/org/parosproxy/paros/common/AbstractParam.java

```diff
@@ -34,6 +34,7 @@
 // ZAP: 2019/06/01 Normalise line endings.
 // ZAP: 2019/06/05 Normalise format/style.
 // ZAP: 2020/11/26 Use Log4j 2 classes for logging.
+// ZAP: 2022-09-08 Use format specifiers instead of concatenation when logging.
```

Comment:
btw, it's using a different date format (not that it matters).

Reply:
Oops, I can tweak